app/src/main/res/layout/dialog_share_safe.xml

```diff
@@ -22,6 +22,43 @@
             android:layout_height="match_parent"
             android:padding="24dp">
 
+            <ImageView
+                android:id="@+id/chain_circle"
+                android:layout_width="@dimen/chain_circle_size"
+                android:layout_height="@dimen/chain_circle_size"
+                android:src="@drawable/circle"
+                app:layout_constraintBottom_toBottomOf="@+id/chain_name"
+                app:layout_constraintEnd_toStartOf="@+id/chain_name"
+                app:layout_constraintHorizontal_bias="0.5"
+                app:layout_constraintHorizontal_chainStyle="packed"
+                app:layout_constraintStart_toStartOf="parent"
+                app:layout_constraintTop_toTopOf="@+id/chain_name"
+                app:tint="@color/blue" />
+
+            <ImageView
+                android:id="@+id/link"
+                android:layout_width="24dp"
+                android:layout_height="24dp"
+                android:background="@drawable/background_selectable"
+                android:src="@drawable/ic_link_green_24dp"
+                app:layout_constraintBottom_toBottomOf="@id/safe_address"
+                app:layout_constraintEnd_toEndOf="parent"
+                app:layout_constraintStart_toEndOf="@id/safe_address"
+                app:layout_constraintTop_toTopOf="@id/safe_address" />
+
+            <ImageView
```

Comment:
The outline looks incomplete:

![image](https://user-images.githubusercontent.com/246473/123665178-502fa500-d838-11eb-8e0e-9268e57ac9e7.png)